src/query/expression/src/values.rs

```diff
@@ -713,6 +731,7 @@ impl PartialOrd for Scalar {
             (Scalar::Variant(v1), Scalar::Variant(v2)) => {
                 jsonb::compare(v1.as_slice(), v2.as_slice()).ok()
             }
+            (Scalar::Geometry(g1), Scalar::Geometry(g2)) => g1.partial_cmp(g2),
```

Comment:
Do Geometry types have order? Direct comparison of binary data may give unexpected results

Reply:
https://postgis.net/docs/manual-1.4/ST_Equals.html

Reply:
> https://postgis.net/docs/manual-1.4/ST_Equals.html

That's eq, not ord.